### PR TITLE
BTAT-2352: Removed audit service and associated files

### PR DESCRIPTION
**Pull request by ja-green**

### CODE REVIEW

File: test/mocks/MockAppConfig.scala [hunk 1]
```diff
@@ -50,9 +50,6 @@ class MockAppConfig(val runModeConfiguration: Configuration, val mode: Mode = Mo
   override val feedbackFormPartialUrl: String = "BasefeedbackUrl"
   override val contactFormServiceIdentifier: String = "VATVC"
   override val staticDateValue: String = "2018-05-01"
-  override val surveyUrl: String = "/some-survey-url"
```
Comment (lewishayqa) [inline test/mocks/MockAppConfig.scala]: App currently does not compile. The relevant values need to be removed from AppConfig, FrontendAppConfig and ConfigKeys.

Comment (macgrewal) [reply test/mocks/MockAppConfig.scala]: @lewishayqa's comments still stand `surveyThankyou` and `surveyUrl` need to be removed from AppConfig, FrontendAppConfig or the tests don't compile. As is the tests are not running.

File: test/mocks/MockAppConfig.scala [hunk 2]
```diff
@@ -50,9 +50,6 @@ class MockAppConfig(val runModeConfiguration: Configuration, val mode: Mode = Mo
...
-  override val surveyThankYouUrl: String = "/some-survey-thankyou-url"
-  override val signOutUrl: String = "/some-gg-signout-url"
```
Comment (macgrewal) [inline test/mocks/MockAppConfig.scala]: signOutUrl should not have been removed, please put this back in.

File: conf/application.conf [hunk 3]
```diff
@@ -140,11 +140,6 @@ government-gateway {
   host = "http://localhost:9025"
 }
 
-feedback-survey-frontend {
```
Comment (lewishayqa) [inline conf/application.conf]: This config needs to be put back in as there are still values in frontendAppConfig relying on it (surveyUrl, signOutUrl).

**Pull request merged**